readme.md

```diff
@@ -41,7 +41,10 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+There is only one body element in traditional html so I would use querySelector.  
+
+var theBody= document.querySelector("body");
+theBody.style.backgroundColor = "red";
 
```

Comment:
additionally `.querySelectorAll` returns an array of dom elements so it would need to be indexed.